migration/servicedmigration/service.py

```diff
@@ -0,0 +1,39 @@
+
+import sys
+import json
+
+from version import versioned
+
+
+class Service:
+    """
+    Manages loading, altering, and commiting services.
+    """
+
+    @versioned
+    def __init__(self):
```

Comment:
Good idea. Any objection to a wrapper function that handles making a Service from argv[] for the user-developer? I'd like to avoid boilerplate.